common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -4018,6 +4022,32 @@ object CelebornConf extends Logging {
       .bytesConf(ByteUnit.BYTE)
       .createWithDefaultString("64m")
 
+  val CLIENT_PUSH_SORT_USE_ADAPTIVE_MEMORY_THRESHOLD: ConfigEntry[Boolean] =
+    buildConf("celeborn.client.spark.push.sort.memory.useAdaptiveThreshold")
+      .withAlternative("celeborn.push.sortMemory.useAdaptiveThreshold")
+      .categories("client")
+      .doc("adaptively adjust threshold for sort shuffle writer's memory threshold")
+      .version("0.5.0")
+      .booleanConf
```

Comment:
Is the threshold numberic? IMO, the threshold should not be boolean.

Reply:
it is the switch of whether to turning on the adaptive feature